lib/openstax/auth/strategy_2.rb

```diff
@@ -0,0 +1,92 @@
+require 'json'
+require 'json/jwt'
+require 'active_support'
```

Comment:
I think you don’t need the require on line 3.  Just using the one on line 4 keeps dependencies down.